### DISCUSSION TITLE
Upgrade from Omega Launcher

### DISCUSSION THREAD

**Discussion by 2br-2b** [otakuhqz/Omega#253] [General]
Hi,

I'm currently a happy user of Omega Launcher (version 0.8.1, build 280).

I didn't realize that Omega Launcher had rebranded and changed to Neo Launcher. When I try to update the app via Droid-ify (an F-droid client), however, I receive this notice:
> This version is signed with a different certificate than the one installed on your device. Uninstall that first.

As I wasn't keeping up with the project at the time, I didn't know about this shift.

Looking at the release notes for [version 0.9.0](https://github.com/NeoApplications/Neo-Launcher/releases/tag/0.9.0) (when I assume this shift occurred), I see two notes that give me worry:
- Note 1: incompatible with backups from older versions
- Note 2: missed features from older versions may be added at later releases

So, if I were to upgrade, it seems that:
1. I would redo my desktop and configuration from scratch;
2. Some features might be missing from the current version.

I don't want to upgrade and lose all my changes, nor do I want to lose access to any features that I currently use. In addition, I want to make sure that when the code rebases to Android 13 soon, I don't need to restart my configuration again.

On the other hand, it's always good to have updates. Security updates are essential, and I don't want to miss out on any new features implemented into Neo Launcher.

Should I backup Omega Launcher, delete it, install Neo Launcher, and then try to restore the backup? Or, on the other hand, should I keep using Omega Launcher (it works fine for what I need it to do) and just not update?

**Comment by machiav3lli** [comment]
I think you already have the answer to your question: As said in Note 1, backups aren't compatible as the project has undergone a total overhaul.

You may miss some features at first but many other features have been added e.g. extensive Gestures (still available only in test builds, available in the community groups) or improved e.g. Dash. The overhaul was much needed to be able to ensure sustainability of the project and easy edition of features on the long term and backward compatibility had a high cost factor which we prefer to invest in future development.

**Reply by 2br-2b** [reply]
Thanks for the quick response!

Just to be sure then: I shouldn't lose all of my customizations when future rebases happen?

**Reply by machiav3lli** [reply]
We certainly have that in mind, but we can never be 100% sure, there's too many factors that play a rule, most importantly weighing effort and result.
